### PR TITLE
Document list | dropdown paddings

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #1739 and #1809

## AC
- [x] Verify that both Categories and Issues dropdown have 14px padding for each option while Clear category filiter and Clear tag filter have 14px margin.

***Testing***
UI Verified

***ScreenShot***
padding

### CODE REVIEW

File: app/assets/stylesheets/reader/_document_list.scss [hunk 1]
```diff
@@ -208,6 +210,8 @@ $tag-backgrond-color: #d6d7d9;
 
       li {
         margin-bottom: 0;
+        padding-top: 14px;
```
Comment (mdbenjam) [inline app/assets/stylesheets/reader/_document_list.scss]: There is too much padding now. The label adds 10px already. I think we should be adding padding to that directly, maybe?

File: app/assets/stylesheets/reader/_document_list.scss [hunk 2]
```diff
@@ -288,6 +291,7 @@ $tag-backgrond-color: #d6d7d9;
       .cf-category-name {
         line-height: 1;
         padding-left: 7px;
+
```
Comment (mdbenjam) [inline app/assets/stylesheets/reader/_document_list.scss]: delete the newline

File: app/assets/stylesheets/reader/_document_list.scss (hunk 1 above)
Comment (Sjones352) [inline app/assets/stylesheets/reader/_document_list.scss]: I decided to remove the padding on the label and added it to li tag.

File: app/assets/stylesheets/reader/_document_list.scss (hunk 2 above)
Comment (Sjones352) [inline app/assets/stylesheets/reader/_document_list.scss]: Deleted

File: app/assets/stylesheets/_main.scss [hunk 3]
```diff
@@ -519,3 +519,8 @@ select {
   }
 }
 
+// add to commons
+.cf-form-checkboxes {
```
Comment (mdbenjam) [inline app/assets/stylesheets/_main.scss]: This messes up the checkboxes on the pdf viewer page:
screen shot 2017-07-07 at 3 44 33 pm

Comment (Sjones352) [inline app/assets/stylesheets/_main.scss]: Fixed it.

File: app/assets/stylesheets/_main.scss [hunk 4]
```diff
@@ -518,4 +518,3 @@ select {
     margin-bottom: 8px;
   }
 }
-
```
Comment (mdbenjam) [inline app/assets/stylesheets/_main.scss]: We should add this newline back. Don't need to change this file.

File: app/assets/stylesheets/reader/_document_list.scss [hunk 5]
```diff
@@ -184,19 +180,21 @@ $tag-backgrond-color: #d6d7d9;
       padding-bottom: 0;
       margin: 0;
       max-height: 325px;
-      overflow-y: scroll;
```
Comment (mdbenjam) [inline app/assets/stylesheets/reader/_document_list.scss]: Why are we removing these?

Comment (Sjones352) [inline app/assets/stylesheets/reader/_document_list.scss]: Talked to Gina and she suggested that we remove the scroll.

File: app/assets/stylesheets/_main.scss (hunk 4 above)
Comment (Sjones352) [inline app/assets/stylesheets/_main.scss]: Added Back

File: client/app/reader/DocTagPicker.jsx [hunk 6]
```diff
@@ -27,13 +27,13 @@ TagSelector.propTypes = {
 const DocTagPicker = ({ tags, tagToggleStates, handleTagToggle }) => {
   return <ul className="cf-document-filter-picker cf-document-tag-picker">
     {tags.map((tag, index) => {
-      return <div className="cf-category-selector" key={index}>
+      return <li><div className="cf-category-selector" key={index}>
```
Comment (mdbenjam) [inline client/app/reader/DocTagPicker.jsx]: Wow I'm surprised this was a div and not an li to begin with. Nice catch. Can you remove the div? Just move the class into the li.

Comment (Sjones352) [inline client/app/reader/DocTagPicker.jsx]: Fixed

File: app/assets/stylesheets/_main.scss (hunk 4 above)
Comment (mdbenjam) [inline app/assets/stylesheets/_main.scss]: I don't see that. Still looks like it's removed.

**Pull request merged**